github/util.go

```diff
@@ -21,9 +21,10 @@ import (
 	"fmt"
 	"net/http"
 
-	gitprovider "github.com/fluxcd/go-git-providers"
-	"github.com/fluxcd/go-git-providers/validation"
 	"github.com/google/go-github/v32/github"
+
```

Comment:
is this newline expected (for all of these files)?

Reply:
Yes, I wanted to separate the internal packages from all the others.

Reply:
ok